builder-buildpackless.toml

```diff
@@ -0,0 +1,10 @@
+description = "Ubuntu 24.04 Noble Numbat base image with no buildpacks included. To use, specify buildpacks at build time."
+
+[lifecycle]
+  version = "0.20.7"
+
+[stack]
+  build-image = "docker.io/paketobuildpacks/ubuntu-noble-build-base:0.0.2"
```

Comment:
@pacostas Shouldn't the build image be called `ubuntu-noble-build` since there is only one variant?

Reply:
Related PR https://github.com/paketo-buildpacks/ubuntu-noble-base-images/pull/10

Reply:
@c0d1ngm0nk3y I think we are good now with the images.